### PR TITLE
*: fix markdownlint issues in all docs

### PR DESCRIPTION
**Pull request by yikeke**
### What is changed, added or deleted? 

fix markdownlint issues in all docs:
1. remove extra space at the end of a line
2. fix consecutive blank lines
3. others

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs-cn/pull/1494

### Which version does your change affect? 

all versions

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ name: "\U0001F41B Bug Report"
 about: Typo, Grammatical error, etc.。
 ---
 
+<!-- markdownlint-disable MD012 -->
+<!-- markdownlint-disable MD029 -->
+<!-- markdownlint-disable MD041 -->
```
Comment (disksing) [inline .github/ISSUE_TEMPLATE/bug-report.md]: I think it would be better to skip `.github` directory in CI.

Comment (aylei) [reply .github/ISSUE_TEMPLATE/bug-report.md]: Of course, I will fix this

Comment (yikeke) [reply .github/ISSUE_TEMPLATE/bug-report.md]: ok, @aylei will fix this.

Comment (aylei) [reply .github/ISSUE_TEMPLATE/bug-report.md]: This directory is indeed skipped in the CI

Comment (yikeke) [reply .github/ISSUE_TEMPLATE/bug-report.md]: He said that currently the markdownlint check does skip `.github` directory now. I will remove these three notes from `.github`.

Comment (yikeke) [reply .github/ISSUE_TEMPLATE/bug-report.md]: Fixed it in https://github.com/pingcap/docs-cn/pull/1611.

File: .github/ISSUE_TEMPLATE/question.md [hunk 2]
```diff
@@ -19,6 +19,6 @@ Before asking a question, make sure you have:
 - Searched open and closed [GitHub issues](https://github.com/pingcap/docs-cn/issues?utf8=%E2%9C%93&q=is%3Aissue).
 <!-- - 查阅了相关的 issues。 -->
 - Read the documentation:
+    - [TiDB Doc](https://github.com/pingcap/docs)
```
Comment (lilin90) [inline .github/ISSUE_TEMPLATE/question.md]:
```suggestion
    - [TiDB Docs](https://github.com/pingcap/docs)
```

File: .github/ISSUE_TEMPLATE/question.md [hunk 3]
```diff
@@ -19,6 +19,6 @@ Before asking a question, make sure you have:
...
+    - [TiDB Doc-cn](https://github.com/pingcap/docs-cn)
```
Comment (lilin90) [inline .github/ISSUE_TEMPLATE/question.md]:
```suggestion
    - [TiDB Docs-cn](https://github.com/pingcap/docs-cn)
```

**Pull request merged**